chart/values.yaml

```diff
@@ -1 +1,12 @@
 extraPorts: []
+
+coreDnsOverrides: |
+  rewrite stop {
+    name regex (.*\.admin\.uds\.dev) admin-ingressgateway.istio-admin-gateway.svc.cluster.local answer auto
+  }
+  rewrite stop {
+    name regex (.*\.uds\.dev) tenant-ingressgateway.istio-tenant-gateway.svc.cluster.local answer auto
+  }
+  rewrite stop {
+    name regex (.*\.uds\.dev) host.k3d.internal answer auto
+  }
```

Comment:
Definitely a nit but I don't think this final rewrite (`host.k3d.internal `) will ever be hit since the previous rewrite for tenant has the same regex + `stop` behavior.

Reply:
I was thinking that the previous one _could_ fail if no Istio gateways actually exist, in the case where this is not deployed with UDS Core. I could be wrong though, and I guess when would this be used outside of at least slim-dev being deployed...?

Reply:
Oh wait, my bad, the stop condition is based on a hit of the regex being validated, so even if the rewrite fails when it goes to the gateway, CoreDNS doesn't care and would never go to the next hit anyway.

Reply:
Fixed!